code/modules/ruins/syndicate_space_base.dm

```diff
@@ -57,7 +57,8 @@
 		/obj/item/ammo_box/magazine/m10mm = 1,
 		/obj/item/flashlight/seclite = 1,
 		/obj/item/clothing/mask/gas/syndicate = 1,
-		/obj/item/tank/internals/emergency_oxygen/engi/syndi = 1
+		/obj/item/tank/internals/emergency_oxygen/engi/syndi = 1,
+		/obj/item/stack/spacecash/c100 = 1
```

Comment:
not a big deal, but from a gameplay standpoint, this 100 space credits will pretty much go unused in all cases. Especially when there's already 2k on the new base.

Reply:
for some reason i cannot figure out how to make claw machine free, so i give syndicate 100 credits to play it.